src/main/java/org/apache/sling/jcr/packageinit/impl/ExecutionPlanRepoInitializer.java

```diff
@@ -163,6 +163,9 @@ public void processRepository(SlingRepository slingRepository) throws Exception
                         builder.with(session);
                         ExecutionPlan xplan = builder.execute();
                         if (xplan.getTasks().size() > 0) {
+                            if (xplan.hasErrors()) {
+                                throw new IllegalStateException("Excecutionplan execution contained errors - cannot complete repository initialization.");
```

Comment:
It would be good to include the errors from the tasks in the exception. That way debugging is way easier.